src/base/bittorrent/torrenthandle.cpp

```diff
@@ -894,11 +891,12 @@ bool TorrentHandle::hasFilteredPieces() const
     return false;
 }
 
-int TorrentHandle::queuePosition() const
+int TorrentHandle::queuePosition(const DataRelevance relevance) const
```

Comment:
One thing I don't like here (I made it myself initially) is to have it in the TorrentHandle interface. Now I think it should stay inside the implementation.
So it would be better to revert changes in this method and use `nativeHandle().queue_position()` in cases we need actual data.

Reply:
Let me know if that's ok.

Reply:
Looks OK for me.